### ISSUE TITLE
Path to File

### ISSUE THREAD

**Issue by kookma** [wincentbalin/tiddlywiki-file-link-import#1]
Great tool! 
It does not keep the path! So, the link does not work except you put all files in the same directory as html file.

There is a `_canonical_uri field in Tiddlywiki allows you to import files but only a link to them.

Also, you may have a look at file upload from Saq Imtiaz

**Comment by wincentbalin** [comment]
The [HTML standard](https://html.spec.whatwg.org/multipage/dnd.html#dom-datatransfer-files-dev) states that for files transferred via drag-and-drop only the file name, without any path information, is provided. So yes, both the wiki _and_ the files imported with this tool _must_ be in the same directory.

Using `_canonical_uri` adds some peculiarities I wanted to avoid and thus I did not use it.

I looked at the Saq Imtiaz's plug-in and it has a different scope. My intention is to store a wiki file with media (or any other) files in the same directory. This will work even if opening the wiki as a file on a computer without Internet connection. The plug-in you mentioned loads files into cloud storages. Nothing wrong with that, but this would not work in my use case.

Thank you very much for the feedback!
